getting-started/lambda-edge.md

```diff
@@ -0,0 +1,157 @@
+# Lambda@Edge
+
+[Lambda@Edge](https://aws.amazon.com/lambda/edge/) is a serverless platform by Amazon Web Services. It allows you to run Lambda functions at Amazon CloudFront's edge locations, enabling you to customize behaviors for HTTP requests/responses.
+
+Hono supports Lambda@Edge with the Node.js 18+ environment.
+
+## 1. Setup
+
+When creating the application on Lambda@Edge,
+[CDK](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-cdk.html)
+is useful to setup the functions such as CloudFront, IAM Role, API Gateway, and others.
+
+Initialize your project with the `cdk` CLI.
+
+```
+mkdir my-app
+cd my-app
+cdk init app -l typescript
+npm i hono
+mkdir lambda
+```
+
+## 2. Hello World
+
+### Origin
```

Comment:
Is 'Origin' necessary? I think just the `hono/lambda-edge` example would be better.

Reply:
fixed.